### ISSUE TITLE
0.75.0 Ubuntu: Cursor mode is missing

### ISSUE THREAD

**Issue by RobbingDaHood** [sedwards2009/extraterm#436]
I just installed version [extratermqt_0.75.0_amd64.deb](https://github.com/sedwards2009/extraterm/releases/download/v0.75.0/extratermqt_0.75.0_amd64.deb) on Ubuntu 23.10, I am using the fish terminal. 

I sourced the setup for fish and frames are showing up. [extraterm-commands-0.75.0.zip](https://github.com/sedwards2009/extraterm/releases/download/v0.75.0/extraterm-commands-0.75.0.zip)

![image](https://github.com/sedwards2009/extraterm/assets/16130319/a840f78c-74cf-4494-9c21-d06c369edc32)

I am just not able to see any cursor mode.  Here is a picture of me searching for it without any luck: 
![image](https://github.com/sedwards2009/extraterm/assets/16130319/868d300b-985a-4137-bb03-91798a735b9d)
 
The guide mentions a cursor mode but not how to enter it: https://extraterm.org/guide.html

My command palette does now show anything when I search for "cursor" either. 

How do I enable the "cursor mode"?

**Comment by sedwards2009** [comment]
In the last two years I've moved Extraterm from being based on Electron to Qt. The cursor mode is one of the things which did not make the transition. I've spent some time recently on updating the guide and documentation, but I haven't updated that part about the cursor mode. Sorry about any confusion or disappointment there.

The version which did include this cursor mode is: https://github.com/sedwards2009/extraterm/releases/tag/v0.59.4

Putting a cursor mode back into the Qt version of Extraterm isn't a high priority, but I am interested in hearing from anyone who likes it.
